<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Devendra" failures="2" tests="2" name="Default test(failed)" time="0.2" errors="0" timestamp="2023-05-02T18:37:35 IST">
  <testcase classname="RetryLoginDemo.TestCasesExample" name="TestCase02" time="0.047">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryLoginDemo.TestCasesExample.TestCase02(TestCasesExample.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestCase02 -->
  <testcase classname="RetryLoginDemo.TestCasesExample" name="TestCase01" time="0.014">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryLoginDemo.TestCasesExample.TestCase01(TestCasesExample.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestCase01 -->
  <testcase classname="RetryLoginDemo.TestCasesExample" name="TestCase03" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase03 -->
</testsuite> <!-- Default test(failed) -->
